src/main/domain/Logger.js

```diff
@@ -0,0 +1,38 @@
+export class Logger {
+  constructor({level, key}) {
+    this._key = key
+    this._level = level
+    this._console = console
+  }
+
+  debug(log) {
+    if (this._level.shouldLog('debug')) {
+      this._log({
+        level: this._level.getLabel('debug'),
+        writter: this._console.log,
+        log
+      })
+    }
+  }
+
+  error(log) {
+    if (this._level.shouldLog('error')) {
```

Comment:
ℹ️ you're invoking a `Level` method with a `'error'` key that is really owned by the Level class, would be better to have if `(this._level.shouldLogError())` instead?

Reply:
You are right, I've changed it. Now it has more data context consistency but the line of code have increased considerably